client/src/lib/repositories/datapass.ts

```diff
@@ -0,0 +1,50 @@
+import type { Fetch } from "src/definitions/fetch";
+import { getApiUrl } from "../fetch";
+
+type DatapassUserPayload = {
+  siret: string;
+  email: string;
+  token: string;
+};
+
+type CreatedDatapassUserResponse = {
+  id: string;
+  siret: string;
+  email: string;
+  role: string;
+  apiToken: string;
+};
+
+type CreateDatapassUser = (opts: {
+  fetch: Fetch;
+  data: DatapassUserPayload;
+}) => Promise<CreatedDatapassUserResponse>;
```

Comment:
Je séparerais le `token` dans un autre paramètre car c'est un détail d'authentification, il n'a pas le même statut que email ou organizations, qui sont vraiment des donneés qui feront partie du user. Je pense qu'il faut donc le traiter comme on traite `apiToken`.

```suggestion
type CreateDatapassUser = (opts: {
  fetch: Fetch;
  token: string;
  data: DatapassUserPayload;
}) => Promise<CreatedDatapassUserResponse>;
```